src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -5167,6 +5175,42 @@ namespace EnergyPlus::OutputReportTabular {
         print(state.files.audit, variable_fmt, "numCompSizeTableEntry", state.dataOutRptPredefined->numCompSizeTableEntry);
     }
 
+    bool produceDualUnitsFlags(const int& iUnit_Sys, const iUnitsStyle& unitsStyle_Tab, const iUnitsStyle& unitsStyle_Sql, 
+        iUnitsStyle& unitsStyle_Cur, bool& produce_Tab, bool& produce_Sql)
+    {
+        // January 2021:
+        // PURPOSE OF THIS SUBROUTINE:
+        // This routine sets the order and flags for tabular data and sqlite writing for a Dual-Unit reporting. 
+        // It will set the unit styles to used, a flag for tabular data writing, and a flag for sqlite writing.
+        // The function will return a false flag if only one round of writing is needed 
+        // and will return a true flag if a second round sqlite writing is not needed.
```

Comment:
I'm very confused by this last sentence.  What is true and what is false?

Reply:
Thanks for spotting this. This is now corrected.